content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,78 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [--targets [<path> [<path> ...]]]
+                      [-t <name_or_path>] [-x <field>] [-y <field>]
+                      [--no-csv-header] [--title <text>]
```

Comment:
```suggestion
                      [--no-header] [--title <text>]
```
per https://github.com/iterative/dvc/pull/4001

Reply:
Thanks for the complete and specific suggestions Ruslan 🙂 

I just grep replaced it in de9110ac though. Resolving all these.